### ISSUE TITLE
Example 5 do not work at all

### ISSUE THREAD

**Issue by fpiette** [CWBudde/PasOpenCL#2]
Example 5 (OpenCL / OpenGL bindings) do not work. I does nothing except displaying a balck screen. I compiled it with Delph 10.3.1 and used the debugger to see what happend. As far as I can see it is the rendering using OpenGL which fails.

**Comment by CWBudde** [comment]
Yes, it happens here as well. However, I undefined GL_INTEROP to make it at least work again. It's not a permanent solution, but a quick workaround.

**Comment by fpiette** [comment]
Thanks for looking at it. In need OpenGL integration actually. I’ll be very happy if you find what is bad in that area
